### ISSUE TITLE
Why Do People with Different Incomes Live in Same Areas?

### ISSUE THREAD

**Issue by khan1792** [uchicago-computation-workshop/luis_bettencourt#3]
For the fist paper (The Statistics of Neighborhoods), I have a question and an opinion/critique that are interrelated. 
It's very interesting that most local-level areas with middle mean income are much more diverse than other areas with high or low mean income. This fact signifies some people with low or high income also live in middle-mean-income areas. **I really want to know if these people are demographically or sociologically different from those who have similar income but live in the areas with high or low mean income.** Since your data source contains socioeconomic information, I believe it's not difficult to answer this question.
My curiosity about this question is derived from the fact that, according to numerous economic and sociological studies, the local-level spatial selection affected by income is a rather normal and rational human behavior. On the contrary, non-selection and non-separation are a little bit abnormal in human society. In case of this, I'm confused by the three following questions. Why do some people who can afford to live in a rich neighborhood (usually with more material and cultural resources) select a middle-mean-income neighborhood? Why do some people with low income choose a place where the living cost is high for them? How can a group of people from different classes get alone well with each other despite their difference in cultural tastes, education, social background, etc? Therefore, my opinion may be different from yours. I believe **the neighborhoods in darker grey demand more local explanations in social science, while the task may be completely different in the perspective of bio-ecological models.** This is why I ask the first question. 
Probably my questions/opinions goes beyond the focus of the first paper, but the results of the paper and the data source can definitely shed light on it. I just feel we have to think about them  if we hope to further understand the inequality and the socioeconomic mechanism of spatial selection. And studying the living modes of the areas with middle mean income may also contribute to the urban policies because we may know how to let people from different classes live together with little conflicts.

**Comment by lmbett** [comment]
Agreed, the additional explanation depends on what you already know, you can come "from the top down" or "from the bottom up".  Other variables (age, education, race, ...) exist in this data but we did not have time to explore them all yet ... would be nice to do.
